build.gradle

```diff
@@ -147,13 +148,17 @@ project ('spring-kafka') {
 
 	dependencies {
 		compile "org.springframework:spring-messaging:$springVersion"
-		compile "org.apache.kafka:kafka-clients:$kafkaVersion"
+		compile ("org.apache.kafka:kafka-clients:$kafkaVersion") {
+			exclude group: 'org.slf4j'
+		}
 		compile ("com.fasterxml.jackson.core:jackson-core:$jacksonVersion", optional)
 		compile ("com.fasterxml.jackson.core:jackson-databind:$jacksonVersion", optional)
 
 		testCompile project (":spring-kafka-test")
 		testCompile "org.assertj:assertj-core:$assertjVersion"
 		testCompile "org.springframework:spring-tx:$springVersion"
+
+		runtime ("org.slf4j:slf4j-jcl:$slf4jVersionRuntime", optional)
```

Comment:
This is `runtime` dependency, so it won't be included to the final POM for the artifact. Therefore `optional` is redundant.
Will fix on merge.